src/ui/molecules/data-table/index.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import styled, { StyledComponent } from 'styled-components';
+import { Table, Tbody, Td, Th, Thead, Tr } from 'ui/atoms';
+import { Variant } from 'lib/types';
+
+interface ColumnProps {
+  property: string;
+  title: React.ReactNode | string;
+}
+
+interface DataTableProps {
+  className: string;
+  columns: Array<ColumnProps>;
+  placeholder: string;
+  values: Array<Record<string, string | number | null>>;
+}
+
+export const DataTableBase: React.FC<DataTableProps & Variant> = ({
+  className,
+  columns,
+  placeholder = '----',
+  values,
+  variant = 'secondary',
+}) => (
+  <Table columns={columns.length} variant={variant} className={className}>
+    <TableHead head={columns} />
+    <Tbody>
+      {values.map(({ id, ...row }) => (
+        <Tr key={id}>
+          {columns.map(({ property }) => (
+            <Td key={property}>{row[property] || placeholder}</Td>
+          ))}
```

Comment:
```suggestion
          {columns.map(({ property, cell }) => {
          	const Cell = isComponent(cell) ? cell : CellContentDefault;
          	return (
            	<Td key={property}>
            		<Cell
            			property={property}
          				value={row[property]}
          				placeholder={placeholder}
      				/>
            	</Td>
          	)
          })}
```

А здесь тогда тоже можно будет создать свой рендерер